### PR TITLE
[POZ-2020] Initial updates

### PR DESCRIPTION
**Pull request by jakubjakubik**
Initial configuration for the Poznan 2020 DevOpsDays

### CODE REVIEW

**Comment by jakubjakubik** [comment]
@mazek

File: data/events/2020-poznan.yml [hunk 1]
```diff
@@ -100,10 +100,13 @@ team_members: # Name is the only required field for team members.
     employer: "Mirantis"
     twitter: "eftepede"
     image: bartek-stalewski.jpg
-  - name: "Klaudia Jasniak"
+  - name: "Klaudia Nawrot"
     employer: "JCommerce"
-    linkedin: "https://www.linkedin.com/in/klaudia-ja%C5%9Bniak-97283180/"
+    linkedin: "https://www.linkedin.com/in/klaudia-nawrot-ja%C5%9Bniak-97283180/"
     image: klaudia-jasniak.jpg
```
Comment (adrianmoisey) [inline data/events/2020-poznan.yml]: I assume this should be changed to `klaudia-nawrot.jpg` ?

```suggestion
    image: klaudia-nawrot.jpg
```

**Comment by adrianmoisey** [comment]
Can you please email info@devopsdays.org with the details of your new organiser (Artur). We need full name and email address, to be added to your mailing list and to slack.

**Comment by jakubjakubik** [comment]
@adrianmoisey On 2nd Oct @mazek emailed the updated list and @serhatcan processed it. Artur is on that list. do we need to do anything more?

**Comment by serhatcan** [comment]
@jakubjakubik looks like you added a new organizer "Artur Szott" and updated one of the organizer's last name. We require sending an email to the core with the updated organizer's list so we can update Slack and email listing.

**Comment by serhatcan** [comment]
Let me check the details. If json file and the emailing list don't match, that is my mistake

**Comment by serhatcan** [comment]
Looks like I crossed checked with the json file and because I didn't see Artur's name in the Json file, I skipped it. Because we already have the name in first e-mail and this is fixed, I'll proceed with the updates.

**Comment by serhatcan** [comment]
@adrianmoisey I updated Slack and email list. Anything else?

**Comment by adrianmoisey** [comment]
We're good! Thanks @serhatcan.
Merging this now.

**Pull request merged**